<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestoreSQAnswer</class>
 <widget class="QDialog" name="RestoreSQAnswer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 41, 47);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Enter your phone number to getting the code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Restore secret question answer</string>
   </property>
  </widget>
  <widget class="QLabel" name="CodeError">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
color: rgb(255, 50, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GetCode_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 rgb(0, 200, 250), 
				stop: 0.4 rgb(30, 150, 255),
                stop:1 rgb(60, 90, 255));
	border: 0px;
}
QPushButton:hover
{
	
	background-color: rgb(25, 206, 255);
}</string>
   </property>
   <property name="text">
    <string>Get code</string>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRepeatDelay">
    <number>0</number>
   </property>
   <property name="autoRepeatInterval">
    <number>-4</number>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneNumber_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Phone number</string>
   </property>
  </widget>
  <widget class="QLabel" name="ChooseSecretQuetion_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>475</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Choose the secret question</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SecretQuestion_comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>495</y>
     <width>600</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border: 0px;
background-color: rgb(74, 79, 90);</string>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>The city in which you were born.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>The name of your favorite pet.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>The your favorite food.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>The maiden's name of your mother.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>The make of your first car.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>The model of your first phone.</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Answer_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Answer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Answer_lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>600</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: rgb(74, 79, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeSQ_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>610</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 rgb(0, 200, 250), 
				stop: 0.4 rgb(30, 150, 255),
                stop:1 rgb(60, 90, 255));
	border: 0px;
}
QPushButton:hover
{
	
	background-color: rgb(25, 206, 255);
}</string>
   </property>
   <property name="text">
    <string>Change secret question answer</string>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRepeatDelay">
    <number>0</number>
   </property>
   <property name="autoRepeatInterval">
    <number>-4</number>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Error_window_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>670</y>
     <width>600</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
color: rgb(255, 50, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QFrame" name="UpperBorder_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 66, 75);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Close_button">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{	
	background-color: rgba(0, 0, 0, 0);
	border: 0px; 
}
QPushButton:hover
{ 
	background-color: rgb(249, 22, 34);
 }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-DesktopClient_qt6-Desktop_Qt_6_6_0_MSVC2019_64bit-Debug/ApplicationIcons/CloseCross.png</normaloff>../build-DesktopClient_qt6-Desktop_Qt_6_6_0_MSVC2019_64bit-Debug/ApplicationIcons/CloseCross.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="LeftBorder_frame_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>20</width>
     <height>730</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 66, 75);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="LeftBorder_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>20</width>
     <height>730</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 66, 75);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="BottomBorder_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>760</y>
     <width>740</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 66, 75);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="Code_lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>601</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: rgb(74, 79, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Email_label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneCode_lineEdit">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>270</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: rgb(74, 79, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Choose_your_country_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Choose you country</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Countries_comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>600</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border: 0px;
background-color: rgb(74, 79, 90);</string>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Canada</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Russia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>France</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Germany</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Japan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>China</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
